<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>obsListCreateEditDialogForm</class>
 <widget class="QWidget" name="obsListCreateEditDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>396</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="BarFrame" name="TitleBar">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="stelWindowTitle">
        <property name="palette">
         <palette>
          <active/>
          <inactive/>
          <disabled/>
         </palette>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Observing Lists editor</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="StelCloseButton" name="closeStelWindow">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayoutLineEdit_2">
        <item>
         <widget class="QLabel" name="labelDescription">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>62</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="descriptionLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="12" column="0">
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutCheckBoxRacDecFov">
           <item>
            <widget class="QCheckBox" name="obsListCoordinatesCheckBox">
             <property name="toolTip">
              <string>Add Ra &amp; Dec coordinates</string>
             </property>
             <property name="toolTipDuration">
              <number>3</number>
             </property>
             <property name="text">
              <string>Coordinates (RA and DEC)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="obsListFovCheckBox">
             <property name="toolTip">
              <string>Add field of view</string>
             </property>
             <property name="toolTipDuration">
              <number>3</number>
             </property>
             <property name="text">
              <string>Fov</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutButtons_1">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="obsListAddObjectButton">
             <property name="toolTip">
              <string>Highlight all the objects of the selected list</string>
             </property>
             <property name="toolTipDuration">
              <number>3</number>
             </property>
             <property name="text">
              <string>Add object</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="obsListRemoveObjectButton">
             <property name="toolTip">
              <string>Clear all the highlight objects</string>
             </property>
             <property name="toolTipDuration">
              <number>3</number>
             </property>
             <property name="text">
              <string>Remove object</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="obsListExportListButton">
             <property name="toolTip">
              <string>Export list</string>
             </property>
             <property name="toolTipDuration">
              <number>3</number>
             </property>
             <property name="text">
              <string>Export list</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="obsListImportListButton">
             <property name="toolTip">
              <string>Import list</string>
             </property>
             <property name="toolTipDuration">
              <number>3</number>
             </property>
             <property name="text">
              <string>Import list</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutButtons_2">
           <item>
            <widget class="QPushButton" name="obsListSaveButton">
             <property name="text">
              <string>Save and close </string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="obsListExitButton">
             <property name="text">
              <string>Exit</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="8" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="obsListErrorMessage">
          <property name="styleSheet">
           <string notr="true">QLabel {color : red;}</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="11" column="0">
       <widget class="QTreeView" name="obsListCreationEditionTreeView">
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayoutCheckBoxDefault">
        <item>
         <widget class="QCheckBox" name="obsListDefaultListCheckBox">
          <property name="toolTip">
           <string>Set the list as default</string>
          </property>
          <property name="toolTipDuration">
           <number>3</number>
          </property>
          <property name="text">
           <string>Set list as default</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="obsListLandscapeCheckBox">
          <property name="toolTip">
           <string>Add landscape</string>
          </property>
          <property name="toolTipDuration">
           <number>3</number>
          </property>
          <property name="text">
           <string>Landscape</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayoutLineEdit_1">
        <item>
         <widget class="QLabel" name="labelListeName">
          <property name="text">
           <string>Name of the list:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>39</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="nameOfListLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarFrame</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StelCloseButton</class>
   <extends>QPushButton</extends>
   <header>gui/StelCloseButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
